shared/src/commonMain/kotlin/org/itmodreamteam/myrest/shared/restaurant/RestaurantUpdateInfo.kt

```diff
@@ -0,0 +1,13 @@
+package org.itmodreamteam.myrest.shared.restaurant
+
+data class RestaurantUpdateInfo (
+    val id: Long,
+    val name: String? = null,
+    val description: String? = null,
+    val legalInfo: String? = null,
+    val websiteUrl: String? = null,
+    val phone: String? = null,
+    val email: String? = null, ) {
+    val containsUpdate: Boolean
```

Comment:
vice versa

Reply:
containsUpdate [is not only when] none of the properties is null/blank
containsUpdate == **any** of the properties is **not null/blank**